aptos-move/move-examples/token_objects/ambassador/move/sources/ambassador.move

```diff
@@ -105,6 +105,16 @@ module token_objects::ambassador {
         );
     }
 
+    public entry fun mint_ambassador_token_by_user(
```

Comment:
@junkil-park @movekevin I need to add this entry function, and use in the transaction generator, as otherwise throughput is miniscule (as we can have only 100 transactions per signer pending at any time)

Reply:
OK.